src/axom/spin/ImplicitGrid.hpp

```diff
@@ -1059,12 +1060,13 @@ ImplicitGrid<NDIMS, ExecSpace, IndexType>::QueryObject::visitCandidates(
       continue;
     }
     // currWord now contains the resulting candidacy information for our given point
-    int numBits = axom::utilities::min(bitsPerWord, nbits - (iword * 64));
+    const int numBits =
+      axom::utilities::min(bitsPerWord, nbits - (iword * bitsPerWord));
```

Comment:
Note: I lost way too much time trying to track down this bug! It hardcoded 64-bit words for the underlying `slam::BitSet`!